### PR TITLE
remove dependency on fix

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Review by LuChen-Microsoft — APPROVED** [review]
The change to ACS Chat looks good to me

File: sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_client_async.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 from datetime import datetime
 from uuid import uuid4
 
-import six
```
Comment (annatisch) [inline sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_client_async.py:13]: omg there's actually a pylint:disable for unused imports....

File: sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py [hunk 2]
```diff
@@ -8,7 +8,6 @@
 from datetime import timedelta
 from typing import Any
 import sys
-import six
```
Comment (annatisch) [inline sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py:11]: The email and identity packages didn't have `six` in their setup.py?

Comment (annatisch) [reply sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py:11]: And networktraversal and phonenumbers...

Comment (xiangyan99) [reply sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py:11]: No. None of them list `six` in their setup.py.

**Review by annatisch — APPROVED** [review]

**Pull request merged**